W-13729475 setup desktop duke (#78)
mulesoft/docs-code-builder: modules/ROOT/pages/deploy-a-mule-application-to-cloudhub.adoc

+
image::deployment-icon-location.png[Deploy to CloudHub icon highlighted on top of the editors.]
+

> Additionally, you can open the Command Palette at any time by pressing Ctrl+Shift+p (Windows) or Cmd+Shift+p (Mac), and provide the following command:

Author: ```suggestion
Alternatively, open the Command Palette by pressing Ctrl+Shift+p (Windows) or Cmd+Shift+p (Mac) and providing the following command:
```
